projects/npm-tools/packages/npm-scripts/src/utils/parseBnd.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+
+/**
+ *
+ */
+function parseBnd(filePath = './bnd.bnd') {
+	const content = fs.readFileSync(filePath).toString();
```

Comment:
Pro-Tip™: You can avoid the `toString()` call by changing this to:

```js
const content = fs.readFileSync(filePath, 'utf8');
```

(I think that's the pattern we use in most places in `npm-scripts`, and it's shorter too.)